src/app/pages/preferences/page/forms/general-preferences-form.component.ts

```diff
@@ -144,6 +144,14 @@ export class GeneralPreferencesFormComponent implements OnInit, AfterViewInit, O
           value:prefs.allowPwToggle,
           tooltip: T('This option enables/disables a password toggle button.'),
           class:'inline'
+        },
+        {
+          type: 'checkbox',
```

Comment:
The other checkboxes retain their settings, but this one is a one-time thing.  A button would make it clearer that this action happens once and can't be undone by unchecking the box. We might need to start dividing up the Preferences screen into separate sections like tables, appearance, and so on.  A "Reset customized columns in tables" button could be in the tables section.